guides/RestAPIGuide.md

```diff
@@ -652,30 +652,30 @@ You can also delete any data key/value by performing a DELETE request on the pat
 ### Format
 #### Taxonomy
 The Taxonomy type extends the Entity type, plus the following:
-```
+```json
 {
-  ‡ "readonly" : <boolean>
+  ‡ "readonly" : "<boolean>"
 }
 ```
 
 #### Term
-```
+```json
 {
-  "uuid" : <string>,
-  "parentUuid" : <string>,
-  "term" : <string>,
-  "fullTerm" : <string>,
-  "index" : <int>,
-  "readonly" : <boolean>,
+  "uuid" : "<string>",
+  "parentUuid" : "<string>",
+  "term" : "<string>",
+  "fullTerm" : "<string>",
+  "index" : "<int>",
+  "readonly" : "<boolean>",
```

Comment:
Don't need to quote numbers and booleans in JSON, so we don't need to here I'd say.

Reply:
If they aren't quoted, they render in bright red becasue they are invalid JSON syntax.

Reply:
This would be better reflected as JSON schema, but that would be a more significant change.
https://json-schema.org